src/query/plan/operator.cpp

```diff
@@ -1844,6 +1844,9 @@ class ExpandAllShortestPathsCursor : public query::plan::Cursor {
 
```

Comment:
please make methods out of this pull, it's really hard to debug without having this logic hierarchically separated